setup.cfg

```diff
@@ -6,23 +6,29 @@ addopts =
     --cov=wold2
     --cov-report term-missing
 
+[coverage:run]
+source = wold2
+
+[coverage:report]
+show_missing = true
+
 [compile_catalog]
 directory = wold2/locale
 domain = clld
 statistics = true
 
 [extract_messages]
 add_comments = TRANSLATORS:
-output_file = wold2/locale/wold.pot
+output_file = wold2/locale/wold2.pot
```

Comment:
It may have been unfortunate to call the file `wold.pot` to begin with, but renaming it lateron is typically not a good idea, because the old one does exist after all.

Reply:
Alright, was probably a little bit overeager to put my newfound knowledge about re-creating the translation files into practice ... will change!

Reply:
It seems you didn't run `compile_catalog`, though, because `clld.mo` isn't changed. So all of this would not have had an effect. Maybe the lesson is that apps with translations should have at least one functional test making sure translations work.